polkadot/runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -31,6 +31,7 @@ use crate::{
 	paras,
 	scheduler::{self, FreedReason},
 	shared::{self, AllowedRelayParentsTracker},
+	util::filter_elastic_scaling_candidates,
```

Comment:
We should structure this better in components that make sense. "util" tells nothing and suggests no separation of concerns at all.

Reply:
yeah, I think we'd better move the function here in the paras_inherent module. IMO It's not a util unless it's used by multiple modules